esp/scm/ws_dfu.ecm

```diff
@@ -832,44 +832,87 @@ ESPresponse [exceptions_inline, nil_remove] EraseHistoryResponse
     ESParray<ESPStruct History, Origin> History;
 };
 
-ESPrequest DFUFileAccessRequest
+ESPStruct DFUFileAccessRequestBase
 {
     string Name;
     string Cluster;
+    string JobId;
     int ExpirySeconds(60);
     ESPenum FileAccessRole AccessRole;
     ESPenum SecAccessType AccessType;
-    string JobId;
     bool ReturnJsonTypeInfo(false);
     bool ReturnBinTypeInfo(false);
 };
 
-// DFUPartLocations and DFUPartCopies part of DFUFileAccessResponse below
-ESPStruct DFUPartLocations
+ESPrequest DFUFileAccessRequest
+{
+    EspStruct DFUFileAccessRequestBase RequestBase;
+};
+
+ESPStruct DFUPartLocation
 {
     int LocationIndex;
-    string Location;
+    string Host;
 };
 
-ESPStruct DFUPartCopies
+ESPStruct DFUFileCopy
+{
+    int CopyIndex;
+    int LocationIndex;
+    string Path;
```

Comment:
Just want to check that this is only to be filled in for the DFUFileCreateResponse?